server/src/core/findSelector.ts

```diff
@@ -44,6 +44,12 @@ export default function findSelector(document: TextDocument, position: Position)
         break;
       case TokenType.AttributeName:
         attribute = htmlScanner.getTokenText().toLowerCase();
+
+        // Convert the attribute to a standard class attribute
+        if (attribute === 'classname') {
```

Comment:
does this work with `className` (i.e. camelcased)?

Reply:
In my test cases I had the JSX instance written as "className" and it was converted to "classname" by the scanner.